addons/fleet/models/fleet_vehicle.py

```diff
@@ -203,8 +203,11 @@ def _search_get_overdue_contract_reminder(self, operator, value):
 
     @api.model
     def create(self, vals):
```

Comment:
```suggestion
    def create(self, vals):
        # Fleet administrator may not have rights to create the plan_to_change_car value when the driver_id is a res.user
        # This trick is used to prevent access right error.
        ptc_value = 'plan_to_change_car' in vals.keys() and {'plan_to_change_car': vals.pop('plan_to_change_car')}
        res = super(FleetVehicle, self).create(vals)
        if ptc_value:
            res.sudo().write(ptc_value)
       # rest of the code...
```
```